src/libstd/net/ip.rs

```diff
@@ -1153,6 +1153,28 @@ impl FromInner<c::in6_addr> for Ipv6Addr {
     }
 }
 
+#[cfg(not(stage0))]
+#[stable(feature = "ipv6_u128", since = "1.17.0")]
```

Comment:
You're correct but rustdoc will display the version number on impls that are marked stable and 1.17.0 is clearly wrong for these.